### PR TITLE
[PostgreSql] Restructure test files to speed up daily test pipeline

### PR DESCRIPTION
**Pull request by DaeunYim**
## Description

Submitting a new PR for this with rebased branch.
This is not customer facing changes and only for daily running pipelines for Orcas team.

## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - {Please put the link here}
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by msJinLei** [comment]
The same as MySql

File: src/PostgreSql/readme.md [hunk 1]
```diff
@@ -62,6 +62,9 @@ directive:
   - from: Microsoft.DBforPostgreSQL/preview/2020-02-14-preview/postgresql.json
     where: $
     transform: return $.replace(/\/subscriptions\/\{subscriptionId\}\/resourceGroups\/\{resourceGroupName\}\/providers\/Microsoft\.DBForPostgreSql\/flexibleServers\/\{serverName\}/g, "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DBforPostgreSQL/flexibleServers/{serverName}")
+  - from: swagger-document
```
Comment (msJinLei) [inline src/PostgreSql/readme.md:65]: Remove this line.

**Review by msJinLei — CHANGES_REQUESTED** [review]

**Comment by DaeunYim** [comment]
Made the change as well..

**Comment by DaeunYim** [comment]
Hi! Is there any other comments I can work on?

File: src/PostgreSql/docs/readme.md [hunk 2]
```diff
@@ -8,4 +8,4 @@ This directory contains the documentation of the cmdlets for the `Az.PostgreSql`
 - Packaged: yes
 
 ## Details
-The process of documentation generation loads `Az.PostgreSql` and analyzes the exported cmdlets from the module. It recognizes the [help comments](https://docs.microsoft.com/powershell/module/microsoft.powershell.core/about/about_comment_based_help) that are generated into the scripts in the `..\exports` folder. Additionally, when writing custom cmdlets in the `..\custom` folder, you can use the help comments syntax, which decorate the exported scripts at build-time. The documentation examples are taken from the `..\examples` folder.
\ No newline at end of file
+The process of documentation generation loads `Az.PostgreSql` and analyzes the exported cmdlets from the module. It recognizes the [help comments](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_comment_based_help) that are generated into the scripts in the `..\exports` folder. Additionally, when writing custom cmdlets in the `..\custom` folder, you can use the help comments syntax, which decorate the exported scripts at build-time. The documentation examples are taken from the `..\examples` folder.
```
Comment (msJinLei) [inline src/PostgreSql/docs/readme.md:11]: @DaeunYim please revert the change to this readme.md

**Review by msJinLei — CHANGES_REQUESTED** [review]

File: src/PostgreSql/docs/readme.md (hunk 2 above)
Comment (DaeunYim) [reply src/PostgreSql/docs/readme.md]: sorry missed this,,!! fixed this

**Review by msJinLei — APPROVED** [review]
